src/components/Table/Table.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from "react"
+import { ColumnConfig, ColumnConfigs, WithID } from "./types"
+
+interface Props<Datum extends WithID> {
+  data: Datum[]
+  columnConfigs: ColumnConfigs<Datum>
+}
+
+export function Table<Datum extends WithID>(props: Props<Datum>) {
+  const { data: dataUnsorted, columnConfigs } = props
+  const [sortConfig, setSortConfig] = useState<ColumnConfig<Datum>>()
```

Comment:
Name nitpick ⛏️ 

`sortConfig` threw me off a little bit, I initially thought there was an explicit place for a `SortConfig` and then realized it came from the `ColumnConfig` (as the type suggests).

`sort` is the feature that uses the type `ColumnConfig`, so we could infer a variety of names to represent this:
`sortColumnConfig`, `sortColumn`, `sortConfig`. This last one seems the least obvious to me for some reason, maybe it put me on a mindset of a configuration object, I don't know 🤷🏼

Reply:
Ha ha, yeah—I hear you. I went back and forth on this one. I think the term "config" throws me off a bit. I wonder if it would be odd to simply change the type `ColumnConfig` to `Column`. What do you think?

I struggled with this since a column might be thought of as the one-dimensional Row > Column... but what I intend here is a Table > Column. But the word "config" thrown into the mix doesn't necessarily help anything.